pkg/sql/opt/optbuilder/delete.go

```diff
@@ -41,6 +41,12 @@ func (b *Builder) buildDelete(del *tree.Delete, inScope *scope) (outScope *scope
 	// Find which table we're working on, check the permissions.
 	tab, depName, alias, refColumns := b.resolveTableForMutation(del.Table, privilege.DELETE)
 
+	if tab.IsVirtualTable() {
+		panic(pgerror.Newf(pgcode.ObjectNotInPrerequisiteState,
```

Comment:
"delete from view" seems like a bit of a misleading error message. any reason why we don't say "table" or "virtual table"?